### DISCUSSION TITLE
What is the significance of using proxies with 2captcha?

### DISCUSSION THREAD

**Discussion by galaczi** [berstend/puppeteer-extra#723] [General]
Can somebody please explain on a high level how the recaptcha solver work? If I have a proxy set on the browser level, should I still provide a proxy to 2captcha? Will 2captcha block me somehow if I don't?

**Comment by damodharheadrun** [comment]
No need to use the proxy for the 2captch, becase we are using the third party  services  to resolve the captcha.

for more information go through the  link https://www.npmjs.com/package/2captcha
